nextflow.config

```diff
@@ -0,0 +1,193 @@
+ *  nf-core/deepvariant Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ * Cluster-specific config options should be saved
+ * in the conf folder and imported under a profile
+ * name here.
+ */
+
+ // Global default params, used in configs
+ params {
+
+   container = 'nfcore/deepvariant:dev'
+
+   help = false
+   outdir = 'results'
+   email = false
+
+   //BAM files
+   bam=""
+   bam_folder=""
+   bam_file_prefix=""
+   getBai=false
+
+   // Reference genomes
+   genomes_base = 's3://deepvariant-data/genomes'
+   //genome = 'hg19' //if no fasta specified default of hg19 will be used (see main.nf)
+
+   // Fasta files
+   fasta=false
+   fai=false
+   fastagz=false
+   gzfai=false
+   gzi=false
+
+   // Exome data
+   exome=""
+   bed=""
+
+   // Model
+   modelBase="s3://deepvariant-data"
+   modelFolder="${modelBase}/models"
+   modelName="model.ckpt"
+
+   //Params for the Read Group Line to be added just in case its needed.
+   rgid=4
+   rglb="lib1"
+   rgpl="illumina"
+   rgpu="unit1"
+   rgsm=20
+
+   tracedir = "${params.outdir}/pipeline_info"
+   clusterOptions = false
+   awsqueue = false
+   awsregion = 'eu-west-1'
+   // TODO: Remove this if/when we can. See https://github.com/nextflow-io/nextflow/issues/840
+   pipelineVersion = '1.0dev'
+ }
+
+process.container = 'nfcore/deepvariant:dev'
+
+docker {
+  enabled=true
+}
+
+profiles{
+
+  s3{
+    params.modelFolder="s3://deepvariant-test/models"
+    params.fasta="s3://deepvariant-test/input/ucsc.hg19.chr20.unittest.fasta";
+    params.fai="s3://deepvariant-test/input/ucsc.hg19.chr20.unittest.fasta.fai";
+    params.fastagz="s3://deepvariant-test/input/ucsc.hg19.chr20.unittest.fasta.gz";
+    params.gzfai="s3://deepvariant-test/input/ucsc.hg19.chr20.unittest.fasta.gz.fai";
+    params.gzi="s3://deepvariant-test/input/ucsc.hg19.chr20.unittest.fasta.gz.gzi";
+    params.bam_folder="s3://deepvariant-test/input/";
+  }
+
+  s3wholeGenome{
+    params.fasta="s3://dv-testfiles/hg19.fa";
+    params.fai="s3://dv-testfiles/hg19.fa.fai";
+    params.fastagz="s3://dv-testfiles/hg19.fa.gz";
+    params.gzfai="s3://dv-testfiles/hg19.fa.gz.fai";
+    params.gzi="s3://dv-testfiles/hg19.fa.gz.gzi";
+    params.bam="s3://dv-testfiles/"
+
+  }
+  localtest
+  {
+    params.bam_folder="$baseDir/data";
+    params.fasta="$baseDir/data/hg19.fa";
+    params.fai="$baseDir/data/hg19.fa.fai";
+    params.fastagz="$baseDir/data/hg19.fa.gz";
+    params.gzfai="$baseDir/data/hg19.fa.gz.fai";
+    params.gzi="$baseDir/data/hg19.fa.gz.gzi";
+  }
+  s3g38whole{
+    params.fasta="s3://gencode.38/GRCh38.p10.genome.fa"
+    params.bam_folder="s3://dv-testfiles/g38Bam/"
+  }
+
+  standard {
+  includeConfig 'conf/base.config'
+  includeConfig 'conf/genomes.config'
+  }
+  conda { process.conda = "$baseDir/environment.yml" }
+  docker {
+    docker.enabled = true
+    process.container = params.container
```

Comment:
No need for this line - should already be defined in `base.config` (or this file, currently).